fastchat/train/train_lora.py

```diff
@@ -28,18 +28,20 @@
 from transformers import Trainer, BitsAndBytesConfig, deepspeed
 import torch
 
-from fastchat.train.train import (
+from train import (
```

Comment:
revert this?

Reply:
the original line "from fastchat.train.train import ()" imports the train.py from the fschat package installed from pip. However, I want to import the train.py in this repo since I also modified that file. I assume users have to cd to fastchat.train before running the train_lora script, though.